apps/icelandic-names-registry/backend/src/environments/environment.ts

```diff
@@ -7,7 +7,6 @@ const devConfig = {
   audit: {
     defaultNamespace: '@island.is/icelandic-names-registry',
   },
-  allowedNationalIds: process.env.ALLOWED_NATIONAL_IDS ?? '',
```

Comment:
Should also remove this from icelandic-names-registry/backend/infra file.

Should write a task to remove this from parameter store after the project is released and finished. Same with feature flag.